### PR TITLE
Updated the shop card

### PR DESCRIPTION
**Pull request by agarwalamn**
Changes:

1.  Added flex to the cards to make them responsive 
2.  Changed the card style 
3.  Added on hover animation

Files changed:

1. pages/shop/index.js
2. components/shoplist-card/index.js

### CODE REVIEW

File: components/shoplist-card/index.js [hunk 1]
```diff
@@ -18,31 +18,67 @@ export const ShopCard = (props) => {
...
-        <button onClick={() => handleAddProduct(product.name)}> + </button>
-        <button onClick={() => handleRemoveProduct(product.name)}> - </button>
-        <p> Quantity : {props.quantity}</p>
+        <p className="shop-card-product-name"> {product.name}</p>
+        <p className="shop-card-product-price"> {product.price}</p>
+        <div className="shop-card-product-qnty">
+          <button onClick={() => handleAddProduct(product.name)}> + </button>
+          <span>{props.quantity}</span>
+          <button onClick={() => handleRemoveProduct(product.name)}> - </button>
+        </div>
```
Comment (whyDontI) [inline components/shoplist-card/index.js:27]: Can you please destructure the `props` and `product`.

**Comment by whyDontI** [comment]
You have done UI changes.
Please put some screenshots of Web browser and Mobile.

**Comment by whyDontI** [comment]
Great work @agarwalamn 👍🏽 🔥

File: components/shoplist-card/index.js (hunk 1 above)
Comment (agarwalamn) [reply components/shoplist-card/index.js:27]: should I further destructure the product as it is already destructured from props?

**Review by Harshi7016 — COMMENTED** [review]
Nikhil has covered all , rest looks good to me

File: components/shoplist-card/index.js (hunk 1 above)
Comment (Harshi7016) [reply components/shoplist-card/index.js:27]: (props) can be named as ({product}) , and inside you can destructure for product that looks good in my view

Comment (whyDontI) [reply components/shoplist-card/index.js:27]: You can do something like this,
```
export const ShopCard = ({ product }) => {
  const { name, restOfTheFields } = product;
.
.
.
}
```

Comment (agarwalamn) [reply components/shoplist-card/index.js:27]: we already had this on line number 5
```javascript
  const { product } = props;
```
That is why I was asking

I will convert it to 
```javascript
  const { product, quantity } = props;
```

File: components/shoplist-card/index.js [hunk 2]
```diff
@@ -18,31 +18,67 @@ export const ShopCard = (props) => {
...
-          width: 100px;
-          height: 100px;
+          width: 80%;
+          flex: 1;
         }
         .shop-card__content {
-          padding: 1rem;
           text-align: center;
+          width: 100%;
+          padding: 1em;
```
Comment (whyDontI) [inline components/shoplist-card/index.js:54]: It'll be great if you can go through this blog and make required chages for your task.
https://css-tricks.com/rems-ems/

In case there are any upper level refactoring required which is not related to your task, skip it. We can send a separate PR for those changes.

**Comment by agarwalamn** [comment]
<div>
<p>Desktop preview</p>

<p>Mobile preview</p>

<div>

**Review by whyDontI — CHANGES_REQUESTED** [review]

File: components/shoplist-card/index.js (hunk 1 above)
Comment (whyDontI) [reply components/shoplist-card/index.js:27]: You also need to destructure the `product`

**Comment by agarwalamn** [comment]
Updated it

File: components/shoplist-card/index.js (hunk 1 above)
Comment (Harshi7016) [reply components/shoplist-card/index.js:27]: Good work @agarwalamn

**Review by Harshi7016 — APPROVED** [review]

File: components/shoplist-card/index.js [hunk 3]
```diff
@@ -1,48 +1,85 @@
...
       <style jsx>{`
         .shop-card {
-          display: inline-block;
+          display: flex;
+          flex-direction: column;
+          align-items: center;
+          justify-content: space-between;
           margin: 10px;
           border: 1px solid #eee;
           box-shadow: 0 2px 2px #ccc;
```
Comment (Kratika0907) [inline components/shoplist-card/index.js:37]: We can have box shadow only on hover

File: pages/shop/index.js [hunk 4]
```diff
@@ -55,6 +55,13 @@ const Shop = (props) => {
           font: inherit;
```
Comment (Kratika0907) [inline pages/shop/index.js:55]: The default behavior for font is to inherit it from parent itself, so we can remove this line

**Review by Kratika0907 — CHANGES_REQUESTED** [review]

**Review by whyDontI — APPROVED** [review]
Great work @agarwalamn 👍🏽 🔥

File: pages/shop/index.js (hunk 4 above)
Comment (agarwalamn) [reply pages/shop/index.js:55]: I think that is for the cart button only, I have not made any changes to that button
